code401Challenges/src/main/java/code401Challenges/InsertionSort.java

```diff
@@ -0,0 +1,25 @@
+package code401Challenges;
+
+public class InsertionSort {
+
+    public static void main(String args[]){
+
+    }
+
+    public static void InsertionSort(int [] arr){
+        if(arr.length == 0){
+            throw new IllegalArgumentException("cannot sort empty array");
+        }
```

Comment:
You could have added an extra conditional to check if the length of the array is 1 (no necessary)